docs/configuration.md

```diff
@@ -30,6 +30,12 @@ by passing the `--local` option to the `config` command.
 poetry config virtualenvs.create false --local
 ```
 
+{{% note %}}
+Your local configuration of Poetry application is stored in `poetry.toml` file,
+which is separate from `pyproject.toml`.
+You should not check-in this file into your repository.
```

Comment:
This is not entirely true or enforced. I'd reword it to suggest it is recommended not to commit it as it may contain user specific configuration or sensitive information.

Reply:
reworded!